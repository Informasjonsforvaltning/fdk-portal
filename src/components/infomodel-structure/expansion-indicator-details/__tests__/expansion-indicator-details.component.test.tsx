import React from 'react';
import { cleanup } from '@testing-library/react';
import { shallow } from 'enzyme';

import ExpansionIndicatorDetails from '..';

afterEach(cleanup);

describe('ExpansionIndicatorDetails component', () => {
  it('should with isExpanded like true', () => {
    const container = shallow(<ExpansionIndicatorDetails isExpanded />);

    expect(container).toMatchSnapshot();
  });

  it('should with isExpanded like false', () => {
    const container = shallow(<ExpansionIndicatorDetails isExpanded={false} />);

    expect(container).toMatchSnapshot();
  });
});
