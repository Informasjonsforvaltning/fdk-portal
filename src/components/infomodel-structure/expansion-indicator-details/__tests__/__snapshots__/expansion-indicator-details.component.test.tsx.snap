// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpansionIndicatorDetails component should with isExpanded like false 1`] = `
<styled.button>
  <Styled(SvgIconExpandTextSm) />
  <styled.span>
    Show details
  </styled.span>
</styled.button>
`;

exports[`ExpansionIndicatorDetails component should with isExpanded like true 1`] = `
<styled.button>
  <Styled(SvgIconCollapseTextSm) />
  <styled.span>
    Close details
  </styled.span>
</styled.button>
`;
